standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/utils/MetaStoreServerUtils.java

```diff
@@ -1636,6 +1636,9 @@ public static String getPartitionName(Path tablePath, Path partitionPath, Set<St
 
   public static String getNormalisedPartitionValue(String partitionValue, String type) {
 
+    if (partitionValue.equals("__HIVE_DEFAULT_PARTITION__"))
```

Comment:
I have a quick question here. Does the patch solve the issue regardless of the value of `hive.exec.default.partition.name`? If not, I think it would be better to retreive the default partition name from `HiveConf` rather than using `__HIVE_DEFAULT_PARTITION__`.

Reply:
@ngsg, thanks for reviewing the PR. Yes, the issue is regarding `hive.exec.default.partition.name`. As hive-common is not a dependency in _standalone-metastore/metastore-server/pom.xml_ , i think it better to use MetaConf as done in `getPartitionListByFilterExp` https://github.com/apache/hive/blob/2ff3ed6d83a1633d17bbf5b3c852b34b51548ef8/standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/HiveMetaStoreChecker.java#L270